docker/postgres/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM postgres:10.1
```

Comment:
We are currently stuck on Ubuntu 16.04 Postgres 9.5 so containers should reflect that.

Reply:
alright! switchin' to postgres 9.5 image, which is based on `debian-slim`, I guess that should be okay